examples/karpenter/provisioners/default_provisioner_with_lanch_templates.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: karpenter.sh/v1alpha5
```

Comment:
Typo in the file name `default_provisioner_with_lanch_templates.yaml`

Reply:
updated